holmes/plugins/toolsets/internet/internet.py

```diff
@@ -256,5 +261,5 @@ def __init__(self):
             tags=[
                 ToolsetTag.CORE,
             ],
-            is_default=True,
+            is_default="True",
```

Comment:
Why? not sure i understand...

Reply:
Making type checking green. I don't think it's right to have put that type in `is_default` in the first place but it's there for now.

Reply:
I'll revert this.

Reply:
actually not. There is an issue further up in the file. Probably some code that was never refactored.